src/test/lambda/config/templates.test.ts

```diff
@@ -756,3 +763,80 @@ describe('TemplatesConfigPopulator', async () => {
         })
     })
 })
+
+describe('getExistingConfiguration', async () => {
+    let registry: CloudFormationTemplateRegistry
+    let tempFolder: string
+    let tempTemplateFile: vscode.Uri
+    let tempConfigFile: string
+    let fakeWorkspaceFolder: vscode.WorkspaceFolder
+    let tempConfigFolder: string
+    const matchedHandler = "it'sTheSameHandler"
```

Comment:
nit caps

Reply:
Caps look fine to me?

Reply:
sorry meant capitalize the constant

Reply:
I would consider the `describe` block as akin to a class, so I'm not feeling giving this all-caps. I'm open to being convinced, but this is how we've done it throughout the rest of the codebase.